content/api/commands/mount.md

```diff
@@ -35,6 +35,10 @@ to start with for your commands:
 <template #react>
 
 ```js
+// React 18
```

Comment:
Not sure if we actually want to do this... it seems consistent with the way that we do the Vue/Vue2 examples